### PR TITLE
mod: improve translate RU

### PR DESCRIPTION
**Pull request by or2e**

### CODE REVIEW

File: src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml [hunk 1]
```diff
@@ -4,50 +4,52 @@
...
     <string id="nbOZ9BNX" text="Защитник {NAME} открыл ворота." />
     <string id="bbg3UpPX" text="Начался раунд {ROUND}..." />
     <string id="1y04FNHB" text="Началась волна {WAVE}!" />
     <string id="4HrC30kl" text="Граф был убит!" />
     <string id="tdfOMWmf" text="Защитники были убиты!" />
-    <string id="ck7dhCeM" text="Добро пожаловать, {NAMES}, который только что присоединился к cRPG!" />
+    <string id="ck7dhCeM" text="Добро пожаловать, {NAMES}, кто только что {?IS_PLURAL}присоединился{?}присоединились{\\?} к cRPG!" />
     <string id="FAkcpZdy" text="Персонаж и магазин" />
-    <string id="xAnUFQt2" text="{COUNT} игроков перешли в команду атакующих{newline}" />
+    <string id="xAnUFQt2" text="{COUNT} игроков перешело в команду атакующих{newline}" />
```
Comment (or2e) [inline src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml:45]: I'm missing the 3rd pluralization option here.

`0 игроков`
`1 игрок`
`2 игрока`
`5 игроков` 

I can't rephrase it

File: src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml [hunk 2]
```diff
@@ -4,50 +4,52 @@
...
     <string id="5HdPy5hH" text="Получено золота: {GAIN} (награда: {REWARD}, содержание: {UPKEEP_MESSAGE})." />
     <string id="FQr3PBEQ" text="Потеряно золота: {GAIN} (награда: {REWARD}, содержание: {UPKEEP_MESSAGE})." />
-    <string id="H6BYX9wi" text="Получено компенсации в размере {COMPENSATION} золота за удар по командному товарищу." />
-    <string id="uOMI4EkG" text="Потеряно компенсации в размере {COMPENSATION} золота за удары по командным товарищам." />
-    <string id="2m5CdjYa" text="Вы не смогли позволить себе обслуживание предмета {ITEMS}, в результате чего он сломался и снят. Вам необходимо посетить веб-интерфейс и экипировать более дешевый предмет." />
+    <string id="H6BYX9wi" text="Получена {COMPENSATION} золота в качестве компенсации за урон от союзника по вам." />
+    <string id="uOMI4EkG" text="Потеряно {COMPENSATION} золота в качестве компенсации за урон по союзникам." />
+    <string id="2m5CdjYa" text="Вы не смогли позволить себе ремонт {?IS_PLURAL}предмета{?}предметов{\\?} {ITEMS}, в результате чего он сломался и был снят. Вам необходимо посетить веб-сайт и экипировать более дешевый предмет." />
```
Comment (verdie-g) [inline src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml:13]: `{\\?}` the double slash is only for C# to scape the backslash. `{\?}` in xml.

**Review by verdie-g — APPROVED** [review]
Thanks!

**Review by verdie-g — CHANGES_REQUESTED** [review]
Wrong button

File: src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml (hunk 1 above)
Comment (verdie-g) [reply src/Module.Server/ModuleData/Languages/RU/std_module_strings_xml-rus.xml:45]: Lol I just understood what you meant. Interesting concept. Not sure how TW dealt with that .

**Review by verdie-g — APPROVED** [review]

**Pull request merged**